.ci/jobs/e2e-testing-helm-daily-mbp.yml

```diff
@@ -0,0 +1,44 @@
+---
+- job:
+    name: e2e-tests/e2e-testing-helm-daily-mbp
+    display-name: End-2-End tests for Observability Helm charts Pipeline
+    description: Run E2E Helm Charts test suite daily, including maintenance branches
+    view: E2E
+    project-type: multibranch
+    pipeline-scm:
+      script-path: .ci/e2eTestingHelmDaily.groovy
+      scm:
+        - github:
+            branch-discovery: no-pr
```

Comment:
There is an extra indentation for the whole `scm` section and that's the reason for the pre-commit failure

Reply:
I fixed the indentation problem 👏